sys/net/gnrc/netif/gnrc_netif.c

```diff
@@ -1086,7 +1085,7 @@ static ipv6_addr_t *_src_addr_selection(gnrc_netif_t *netif,
             bf_set(candidate_set, i);
         }
     }
-    /* otherwise apply rule 8: Use longest matching prefix. */
+    /* otherwise apply rule 9: Use longest matching prefix. */
```

Comment:
without looking into detail, i'm confused: Rule 9 relates to Destination Address Selection in RFC 6724. My understanding was that this function is about Source Address Selection.

Reply:
> without looking into detail, i'm confused: Rule 9 relates to Destination Address Selection in RFC 6724. My understanding was that this function is about Source Address Selection.

Yes, totally right. I've reverted that commit.